### ISSUE TITLE
Handlers not found!

### ISSUE THREAD

**Issue by pkhigh** [zhijian-liu/torchprofile#8]
Hi, I was doing profiling on the recently released StarGAN-V2 ( https://github.com/clovaai/stargan-v2 ) pre-trained generator and style_encoder networks. However, certain operations used in these networks are missing from torch profile. Following are the missing handlers:

- "aten::leaky_relu"
- "aten::to"
- "aten::detach"
- "aten::floor"
- "aten::upsample_nearest2d"
- "aten::index"
- "aten::to"
- "aten::stack"

Please suggest where can I add these operations in profile.py to do an accurate profiling?

**Comment by zhijian-liu** [comment]
The profiling result is correct as all these operations you listed do not require MACs. I've updated the package accordingly and the warning should be fixed now. Thanks!
